### PR TITLE
pypi setup - add readme on PyPI

### PR DESCRIPTION
**Pull request by wesinator**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,6 +6,9 @@
 
 VERSION = '2.7.1'
 
+with open('README.rst'), encoding='utf-8') as f:
```
Comment (vEpiphyte) [inline setup.py:9]: This open statement is invalid python syntax.

**Review by vEpiphyte — CHANGES_REQUESTED** [review]
PR needs to be valid syntax

**Comment by wesinator** [comment]
> PR needs to be valid syntax

crap, it was a typo from example. Fixed

File: setup.py [hunk 2]
```diff
@@ -6,6 +6,9 @@
 
 VERSION = '2.7.2'
 
+with open('README.rst', encoding='utf-8') as f:
```
Comment (vEpiphyte) [inline setup.py:9]: @wesinator Would you mind updating this to be conditional, similar to how this is done?
https://github.com/vertexproject/common-tools/blob/master/setup.py#L21

Comment (vEpiphyte) [reply setup.py:9]: and set the type so it is not ambiguous

**Review by vEpiphyte — APPROVED** [review]

**Comment by vEpiphyte** [comment]
Testing this on testpypi - looks nice.

image

**Pull request merged**